docs/source/quickstart.rst

```diff
@@ -10,9 +10,75 @@ Setup
 
 For a more detailed guide on installation options please refer to the :doc:`install` page.
 
+Send/Recv NumPy Arrays
+---------------------
+
+Process 1
+~~~~~~~~~
+
+.. code-block:: python
+
+    import asyncio
+    import ucp
+    import numpy as np
+
+    n_bytes = 2**30
+    host = ucp.get_address(ifname='enp0s25')
```

Comment:
Maybe clarify that this should be the local ethernet interface? It's unlikely that this will be the same for others, changing this to `eth0` probably makes more sense, as that is a more common name.